entries/g/gumlet.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Gumlet": {
+    "domain": "gumlet.com",
+    "url": "https://www.gumlet.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.gumlet.com/docs/profile-overview#password-and-2fa",
+    "categories": [
+      "cloud",
+      "developer",
+      "entertainment"
```

Comment:
```suggestion
```

The entertainment category is for audio and video entertainment, excluding games.